site/specification/ORCv0.md

```diff
@@ -308,7 +308,7 @@ message DateStatistics {
 ```
 
 Timestamp columns record the minimum and maximum values as the number of
-milliseconds since the epoch (1/1/2015).
+milliseconds since the UNIX epoch (1/1/1970 00:00:00 in UTC).
```

Comment:
This is wrong. Timestamps are millis since 1/1/2015, until we added minimumUtc and maximumUtc in ORC-135 f2b8b799c .

Reply:
Thanks for pointing this out! I think ORC-135 belongs to ORCv1 only; therefore I will only fix ORCv1.md and leave ORCv0.md as is.